src/components/Practice/Mechanics6.jsx

```
@@ -0,0 +1,518 @@
+              py: "12px",
+              // background: "transparent",
+              m: 1,
+              textTransform: "none",
+              borderRadius: "12px",
+              border: `1px solid rgba(51, 63, 97, 0.10)`,
+              background: "#FFF",
+              cursor: "pointer",
+              opacity: disabledWords ? 0.25 : 1,
+              pointerEvents: disabledWords ? "none" : "initial",
+            }}
+          >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)